csharp/extractor/Testrunner/Testrunner.cs

```diff
@@ -0,0 +1,90 @@
+using System.Reflection;
```

Comment:
This file feels somewhat odd: shouldn't this live in some bazel specific tooling folder in the internal repo?

Reply:
As we're only using this in the external repo (no C# code in the internal repo), we prefer to host this file in the external repo, too, to avoid synchronized PRs.
If you don't want the file in `csharp/extractor/Testrunner`, we could move it to `misc/bazel/csharp/testrunner` instead. I don't have an opinion on this, but I do think that C# code is best reviewed by someone with C# experience.

Reply:
Let's leave the file where it is. If we're unhappy with it later, we'll move it.